platform/android/java/editor/src/main/java/org/godotengine/editor/GodotEditor.kt

```diff
@@ -110,6 +110,7 @@ open class GodotEditor : GodotActivity() {
 		super.onGodotSetupCompleted()
 		val longPressEnabled = enableLongPressGestures()
 		val panScaleEnabled = enablePanAndScaleGestures()
+		val rotaryInputAxis = getRotaryInputAxis()
```

Comment:
The `rotaryInputAxis` value needs to be passed to the `GodotInputHandler` below on line 121.

Reply:
Fixed.

Reply:
Actually,  I made a mistake; the changes in the `GodotEditor` class are not needed because the editor just used the `Godot` object which applies the setting for it.
So you should be able to revert the changes in this file.